src/components/content/textStyle.js

```diff
@@ -0,0 +1,66 @@
+const TextContentVariant = {
+    short: 'short',
+    paragraph: 'paragraph',
+}
+
+const TextContent = {
+    [TextContentVariant.short]: {
+        base: {
+            display: 'flex',
+            justifyContent: 'center',
+            alignItems: 'center',
+            backgroundColor: 'coral',
+        },
+        onBase: {
+            fontWeight: 'bold',
+        }
+    },
+    [TextContentVariant.paragraph]: {
+        base: {
+            display: 'flex',
+            marginHorizontal: 27,
```

Comment:
all paragraph need to margin horizontal. This is function the layout of container. Not the paragraph itself